index.html

```diff
@@ -2249,6 +2246,34 @@ <h3>
           available underneath an icon).
         </p>
       </section>
+      <section>
+        <h3 id="installation-sec">
+          Launching a web application
+        </h3>
+        <p>
+          The steps to <dfn>launch a web application</dfn> is given by the
+          following algorithm. The algorithm takes a
+          [=Document/processed manifest=] |manifest:processed manifest|,
+          an optional |target URL:URL| and returns a
+          [=service worker client/window client=].
```

Comment:
Below it creates a a new browsing context, not a Service Worker window client.

Reply:
I should update the launch_handler to use browsing context. I went with Service Worker window client in the interest of being compatible with the `clients` interface inside SW but looks like Service Worker window clients are 1:1 with navigated browsing contexts: https://w3c.github.io/ServiceWorker/#control-and-use-window-client

Reply:
Ok, cool. Yeah, it's good to keep this spec SW agnostic.

Reply:
Done: https://github.com/WICG/web-app-launch/pull/81